### ISSUE TITLE
Unable to read components properly

### ISSUE THREAD

**Issue by jippy89** [quasarframework/quasar#5045]
**Describe the bug**
I have a file named `Index.vue` under `src/pages/Home/` folder. I tried calling components from there, but it seems none of them being called. I don't know if this is a problem related to Quasar itself or Webpack.

**Codepen/jsFiddle/Codesandbox (required)**
I tried uploading to Codesandbox but it seems I face some trouble there as well.. While waiting for me to fix that up, I already prepared a [github repo](https://github.com/jippy89/mini-repo) if you want to check the problem

**To Reproduce**
1. Goto the folder, do `npm i`
2. Run `npm run dev`
3. Check console for the error, the error should be something like this..
```
[Vue warn]: Unknown custom element: <OrderPane> - did you register the component correctly? For recursive components, make sure to provide the "name" option.
```
4. The `vue` files that is importing that component is in `src/pages/Home/Index.vue`

**Expected behavior**
The component loads as usual.

**Platform (please complete the following information):**
OS: Manjaro Linux 18.0.4
Node: 11.14.0
NPM: 6.9.0
Yarn: 1.17.3
Browsers: 68.0.1 (64-bit)

**Additional context**
Please do ask for additional context. I tried adding `name` to the component, but it doesn't work.
I was just trying to load a component in the `src/pages/Home/Index.vue` but apparently that doesn't work! I had already declared comments there describing which one that doesn't load. Apparently it's not only the component, I have created a `created` hooks, and it's not functioning as usual.

Thank you.

**Comment by metalsadman** [comment]
components folder has an alias registered, so just refer your files inside it like so `components/yourcomponent.vue`. https://quasar.dev/quasar-cli/cli-documentation/handling-webpack#Webpack-Aliases

**Comment by jippy89** [comment]
Yesss I know that, but I had different file architecture. If you check correctly, I have created `components` folder inside `layout/MainLayout/` and `pages/Index/`, to distinguish whether a component has a special component only to that file. 

And I have been using Quasar for a while, and surprised that this was a problem to me. At first I thought I was just missing something, but after 3 hours not finding it I might be panicked for a while. **In the end,** I don't know what was causing the issue.

I had the same configuration right now. Only installing with `npm`, and the earlier was using `yarn`. I don't know if that's a problem tho'. For now I will close the issue, since I have some work to do. I will report back if I found something later. Thank you all!
